lib/govuk_app_config/govuk_error/configure.rb

```diff
@@ -1,11 +1,13 @@
 GovukError.configure do |config|
+  config.current_environment = ENV["SENTRY_CURRENT_ENV"]
+
   config.before_send = proc { |e|
     GovukStatsd.increment("errors_occurred")
     GovukStatsd.increment("error_types.#{e.class.name.demodulize.underscore}")
     e
   }
 
-  config.silence_ready = !Rails.env.production? if defined?(Rails)
+  config.silence_ready = !Rails.env.production? if Object.const_defined?("Rails")
```

Comment:
Do we need to make this change? I recall it being from an earlier fight with tests that seems to have gone.